src/Core/Command/UploadAvatarHandler.php

```diff
@@ -96,7 +96,10 @@ public function handle(UploadAvatar $command)
         $this->validator->assertValid(['avatar' => $file]);
 
         $manager = new ImageManager;
-        $manager->make($tmpFile)->fit(100, 100)->save();
+
+        $encodedImage = $manager->make($tmpFile)->fit(100, 100)->encode('jpg', 100);
+
+        @file_put_contents($tmpFile, $encodedImage);
```

Comment:
I've looked up the code of `Intervention\Image\Image::save` and it uses `@file_put_contents`. The `save` function will automatically detect the format based on the file extension. So if we pass `.tmp` extension, the format will be `tmp` and that causes the error. The `tempnam` function will add `tmp` extension on Windows servers. If the `save` function has a parameter to manually choose the encoding format, we could use the `save` function.
https://github.com/Intervention/image/blob/master/src/Intervention/Image/Image.php#L122-L152